src/components/ToDoContainer/index.js

```diff
@@ -1,9 +1,143 @@
 import React, { Component } from "react";
 
+
+let id = 0;
+
+class Add extends Component {
```

Comment:
Put each component into its own file and use `import` and `export` to use them in the `TodoContainer` file. File structure generally works like: 

```
ToDoContainer/
- index.js
- styles.js
- components/
--Add/
---index.js
---styles.js
--Delete/
---index.js
---styles.js
```